src/core/system_parameters/beidou_cnav2_ephemeris.cc

```diff
@@ -0,0 +1,185 @@
+/*!
+ * \file beidou_cnav2_ephemeris.cc
+ * \brief  Interface of a BEIDOU CNAV2 EPHEMERIS storage and orbital model functions
+ * \note Code added as part of GSoC 2018 program
```

Comment:
This note should be repeated in all new files that you add, give credit to Google and its economic support